components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -185,6 +185,12 @@ def make_clp_s_command_and_env(
     # fmt: on
 
     if InputType.S3 == clp_config.input.type:
+        aws_access_key_id = clp_config.input.aws_access_key_id
+        aws_secret_access_key = clp_config.input.aws_secret_access_key
+        if aws_access_key_id is None or aws_secret_access_key is None:
```

Comment:
technically, we shouldn't run into this case. However, clp worker will return weird error message if we ever run into a None=aws_access_key_id here, since a env=None is invalid can cause error for subprocess.run.

Just to be safe, I decided to add the guard here.